source/connect/connect-from-the-command-line.txt

```diff
@@ -38,23 +38,47 @@ Command Line Connection Specification
 -------------------------------------
 
 The command line invocation for |compass-short| has two components, the 
-path to the |compass-short| executable and a connection string. The
+path to the |compass-short| executable and a connection string, you can
+optionally pass in the username and password parameters. The
 format is:
 
 .. code-block:: shell
    :copyable: false
- 
-    <path/to/compass/executable> <connection string>
 
-This example is like the basic connection string for a `MongoDB
+    <path/to/compass/executable> <connection string> 
```

Comment:
These should be on one line (or use a \ so that the line is continued despite the carriage return)